src/CMakeLists.txt

```diff
@@ -165,4 +164,53 @@ if(BUILD_EXE)
   set(CPACK_DEBIAN_PACKAGE_MAINTAINER "John Argerus")
 
   INCLUDE(CPack)
+  
+  #grpc build server and client target
+  # Protobuf
+  find_package(Protobuf REQUIRED)
+  message(STATUS "Using protobuf ${Protobuf_VERSION}")
+
+  # Protobuf-compiler
+  set(_PROTOBUF_PROTOC $<TARGET_FILE:protobuf::protoc>)
+
+  # gRPC
+  find_package(gRPC CONFIG REQUIRED)
+  message(STATUS "Using gRPC ${gRPC_VERSION}")
+  set(_GRPC_GRPCPP gRPC::grpc++)
+  set(_GRPC_CPP_PLUGIN_EXECUTABLE $<TARGET_FILE:gRPC::grpc_cpp_plugin>)
+  set(_REFLECTION gRPC::grpc++_reflection)
+  
+  # Include generated *.pb.h files
+  add_library(proto 
+    ../build/src/kuksa.grpc.pb.cc
```

Comment:
referencing build folder is bad, because it will break if you build in another folder. (I.e. cmake totally allows `mkdir mytestbuild; cc mytestbuild; cmake .."

There should be a predefined variable name for the current build folder. Maybe @wenwenchenbosch  knows immediately